### PR TITLE
[Peggy Docs] Whitelisting Docs Update

### PR DESCRIPTION
**Pull request by ElliotFriedman**

### CODE REVIEW

File: smart-contracts/LimitUpdating.md
Comment (musnit) [inline smart-contracts/LimitUpdating.md:5]: can we change this to suggest the same process as we're going with now, ie:
 - create a new whitelist_network_date.json file
 - make a PR
 - get a review/approval
 - get it manually deployed

File: smart-contracts/LimitUpdating.md
Comment (musnit) [inline smart-contracts/LimitUpdating.md:7]: I think this and other env setup will be applicable to other kinds of future tasks too. Can we move this into a general pre-requisite section/document like Runbook.md with a setup section like "Setting up your environment for doing operations work"

File: smart-contracts/LimitUpdating.md
Comment (musnit) [inline smart-contracts/LimitUpdating.md:5]:
```suggestion
Before running the following script go to the data folder and open create a file called `whitelist_<network>_<date>.json`, for example `whitelist_ethereum_feb_21_2021.json`. Change the name of the file to remove the date and insert the current date. Copy the contents from `limitWhitelistUpdate.json` into your newly created file and change the addresses to the address you want to whitelist. Then, update the limit to the desired limit you would like to allow a user to move through the bridge in a single tx. If you want to set this to the max allowable value in solidity, set the limit value to `115792089237316195423570985008687907853269984665640564039457584007913129639935` which is the UINT_MAX in solidity.
```

File: smart-contracts/LimitUpdating.md
Comment (musnit) [inline smart-contracts/LimitUpdating.md:9]:
```suggestion
BRIDGEBANK_ADDRESS='0x30753E4A8aad7F8597332E813735Def5dD395028' truffle exec scripts/bulkSetTokenLockBurnLimit.js --network develop ../data/whitelist_<network>_<date>.json
```

File: smart-contracts/ProdEnvRunbook.md
Comment (musnit) [inline smart-contracts/ProdEnvRunbook.md:5]: fine for now, though would be nice in future if we can use specific variables for each key rather than one private key, eg: OPERATOR_PRIVATE_KEY, PAUSER_PRIVATE_KEY, etc.

**Review by musnit — APPROVED** [review]
a few minor comments, otherwise good to merge once resolved

File: smart-contracts/ProdEnvRunbook.md
Comment (ElliotFriedman) [reply smart-contracts/ProdEnvRunbook.md:5]: The scripts are only designed to be run with one role at once. A single script should not be sending transactions from multiple different addresses.

File: smart-contracts/LimitUpdating.md
Comment (ElliotFriedman) [reply smart-contracts/LimitUpdating.md:7]: Done.

**Comment by banshee** [comment]
I added some minor cleanup suggestions in PR #981, but none of them are necessary to merge this.

**Review by banshee — APPROVED** [review]

**Review by banshee — APPROVED** [review]

**Pull request merged**
